app/src/main/java/org/openimis/imispolicies/CummulativeIndicators.java

```diff
@@ -154,6 +155,10 @@ public void run() {
 
     public void getCommulativeIndicators(String DateFrom, String DateTo){
         try {
+            if (!_General.isNetworkAvailable(this)) {
```

Comment:
Couldn't you just call ca.CheckInternetAvailable() in if?

Reply:
The same in all network validation.

Reply:
Changed to CheckInternetAvailable